packages/mui-material/src/utils/isMuiElement.test.js

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
```

Comment:
This is what remained in mui-material/utils/index.test.js after moving useForkRef and setRef out of this file.